docs/content/configuration/configuring_hostdefiner.md

```diff
@@ -0,0 +1,12 @@
+# Configuring the host definer
+
+Some of the parameters within the HostDefiner custom resource are configurable. Use this information to help decide whether the parameters for your storage system need to be updated.
+
+For more information on using the host definer, see [Using dynamic host definition](../using/using_hostdefinition.md).
+    
+|Field|Description|
+|---------|--------|
+|prefix|Adds a prefix to the hosts defined by the host definer.|
+|connectivity|Selects the connectivity type for the host ports.<br>Possible input values are:<br>- `nvme` for use with NVME over Fibre Channel connectivity<br>- `fc` for use with Fibre Channel over SCSI connectivity<br>- `iscsi` for use with iSCSI connectivity<br><br>By default, this field is blank and the host definer selects the first of available connectivity types on the storage system, according to the following hierarchy: NVMe, FC, iSCSI.|
+|allowDelete|Defines whether the host definer is allowed to delete host definitions.<br>Input values are `true` or `false`.<br>The default value is `true`.|
+|dynamicNodeLabeling|Defines whether the nodes that run our CSI node pod are dynamically labelled or if the user must create the `hostdefiner.block.csi.ibm.com/managed-by=true` label on each relevant node. In addition, this label allows the host definer to find all nodes with this label and manages their storage definition.<br>Input values are `true` or `false`.<br>The default value is `false`, where the user must create each label.|
```

Comment:
```suggestion
|dynamicNodeLabeling|Defines whether the nodes that run the CSI node pod are dynamically labelled or if the user must create the `hostdefiner.block.csi.ibm.com/managed-by=true` label on each relevant node. This label tells the host definer which nodes to manage their host definition on storage.<br>Input values are `true` or `false`.<br>The default value is `false`, where the user must create each label.|
```
AFAIK we never us "our" in official docs to describe the product we develop.
`In addition` makes it sound like a separate thing, when it's actually an elaboration of the first sentence.
`this label allows the host definer to find all nodes with this label` sounds too obvious and is not the main focus.
`storage definition` might sound like a definition of a storage, that is not a host definition

Reply:
done